stock_cycle_count/models/stock_cycle_count.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-18 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class StockCycleCount(models.Model):
+    _name = 'stock.cycle.count'
+    _description = "Stock Cycle Counts"
+    _inherit = 'mail.thread'
+
+    @api.multi
+    def _compute_inventory_adj_count(self):
+        for rec in self:
+            rec.inventory_adj_count = len(rec.stock_adjustment_ids)
+
+    @api.model
+    def create(self, vals):
+        vals['name'] = self.env['ir.sequence'].next_by_code(
+            'stock.cycle.count') or ''
+        return super(StockCycleCount, self).create(vals)
+
+    @api.model
+    def _company_get(self):
+        company_id = self.env['res.company']._company_default_get(self._name)
+        return company_id
+
+    name = fields.Char(string='Name', readonly=True)
```

Comment:
Please follow OCA conventions and set fields definitions above.

Reply:
done